dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+MAINTAINER Madhuri Yechuri <madhuri.yechuri@clusterhq.com>
+
+# Note: apt-get install requires the `-1` (epoch) suffix.
+ENV FLOCKER_VERSION 1.14.0-1
```

Comment:
:question: Does this allow me to override the version at build time?
e.g. `FLOCKER_VERSION=1.15.0.123.aaabbbccc docker build .` 

If so add a short section to the docs explaining how this Dockerfile should be built and run.
 * https://github.com/ClusterHQ/flocker/blob/master/docs/gettinginvolved/infrastructure/packaging.rst